tests/services/error-handling.service.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { ErrorHandlingService } from '../../src/app/core/services/error-handling.service';
+import { FormErrorComponent } from '../../src/app/shared/error-toasters/form-error/form-error.component';
+import { GeneralMessageErrorComponent } from '../../src/app/shared/error-toasters/general-message-error/general-message-error.component';
+import {
+  ERROR_WITH_NO_MESSAGE,
+  ERROR_WITH_NODE_MODULES,
+  HTTP_304_ERROR,
+  HTTP_400_ERROR,
+  HTTP_422_ERROR,
+  HTTP_500_ERROR,
+  OCTOKIT_REQUEST_ERROR,
+  STANDARD_ERROR
+} from '../constants/error.constants';
+
+let errorHandlingService: ErrorHandlingService;
+let mockLoggingService;
+let mockSnackBar;
+
+describe('ErrorHandlingService', () => {
+  beforeEach(() => {
+    mockLoggingService = jasmine.createSpyObj('LoggingService', ['error', 'debug']);
+    mockSnackBar = jasmine.createSpyObj('MatSnackBar', ['openFromComponent']);
+    errorHandlingService = new ErrorHandlingService(mockSnackBar, mockLoggingService);
+  });
+
+  it('should log errors when handling errors', () => {
```

Comment:
Let's group the following specs under `handleError` describe block, since we are only testing this particular method.